src/executables/master_request.c

```diff
@@ -304,6 +304,7 @@ css_process_server_list_info (CSS_CONN_ENTRY * conn, unsigned short request_id)
 	  else
 	    {
 	      required_size += strlen (SERVER_FORMAT_STRING);
+	      required_size += strlen (temp->server_type == SERVER_TYPE_PAGE ? "Page" : "Transaction");
```

Comment:
Please add a variable `std::string server_type_str` and avoid doing `temp->server_type == SERVER_TYPE_PAGE ? "Page" : "Transaction"` multiple times. Also make sure the server_type is really transaction or page.
```suggestion
	      if (temp->server_type == SERVER_TYPE_TRANSACTION)
	        {
	          server_type_str = "Transaction";
	        }
	      else if (temp->server_type == SERVER_TYPE_PAGE)
	        {
	          server_type_str = "Page";
	        }
	      required_size += server_type_str.length ();
```

Same for css_process_all_list_info.